flex/views/experiment/TabSampleSetupView.mxml

```diff
@@ -199,7 +199,7 @@
 			}
 			public function propagateSeqPrepByCore():void {
 				for each (var sample:Object in parentDocument.samples) {
-					sample.@seqPrepByCore = (currentState == 'SolexaSetupState' || (seqPrepRadioGroup.selectedValue !=  null && seqPrepRadioGroup.selectedValue == 'Y')  ? 'Y' : 'N');
+					sample.@seqPrepByCore = (currentState == 'SolexaSetupState'  ? 'Y' : 'N');

```

Comment:
The sample setup tab has propagateSeqPrepByCore and propagateSeqLibProtocol methods that used to depend on questions that are now on a later tab. In simplifying these methods here, this will set "seqPrepByCore" to a default of 'N'. I'm worried that after the other tab sets these values using the additional information, the user coming back to this tab might cause the correct logic to be overwritten.